### PR TITLE
feat: add code block line numbers

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| New feature?     | yes
| Fixed tickets    | comma-separated list of tickets fixed by the PR, if any
| License          | MIT

### Description

- [x] Add code block line numbers, and the default `codeBlockLineNumbers` is `true`.
- [x] Add copy code button

屏幕快照 2019-11-01 下午6 44 35

#### Please, don't submit `/dist` files with your PR!

---
This change is [Reviewable](https://reviewable.io/reviews/marktext/marktext/1572)

### CODE REVIEW

File: src/muya/lib/parser/render/renderBlock/renderCopyButton.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  const iconVnode = h('i.icon', h('i.icon-inner', {
+    style: {
+      background: `url(${copyIcon}) no-repeat`,
+      'background-size': '100%'
+    }
+  }, ''))
+
+  return h(selector, {
+    attrs: {
+      title: 'Copy',
```
Comment (fxha) [inline src/muya/lib/parser/render/renderBlock/renderCopyButton.js]: I prefer `Copy content`.

File: src/muya/lib/eventHandler/resize.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    this.listen()
+  }
+
+  listen () {
+    window.addEventListener('resize', function () {
+      if (this.timer) {
+        clearTimeout(this.timer)
+      }
+      this.timer = setTimeout(() => {
+        console.log('wwww')
```
Comment (fxha) [inline src/muya/lib/eventHandler/resize.js]: Debug code

**Comment by fxha** [comment]
@Jocs There was  still debug code to remove.

**Comment by Jocs** [comment]
>because not documentation exist on master.

You maybe saw an outdated version of files, I already remove them?

**Pull request merged**